.github/workflows/docker.build.yml

```diff
@@ -56,4 +56,19 @@ jobs:
           cache-to: type=local,mode=max,dest=/tmp/.buildx-cache
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
-          labels: ${{ steps.docker_meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.docker_meta.outputs.labels }}
+      #Create PR on deploy repo from a merged PR
+      -
+        name: create pull request for deploy on staging
+        uses: actions/checkout@v3
+        with:
+          repository: rosalieper/wbaas-deploy
+      # Make changes to pull request here
+          run: |
+            yq -i '.image.tag = ${{ steps.docker_meta.outputs.tags }}' k8s/helmfile/env/local/ui.values.yaml.gotmpl
+            yq -i '.image.tag = ${{ steps.docker_meta.outputs.tags }}' k8s/helmfile/env/staging/ui.values.yaml.gotmpl
+      - uses: peter-evans/create-pull-request@v4
```

Comment:
I think there might need to be a `git add` and `git commit` step happening before this one

Reply:
And before you can do that you need to initialise git I think (e.g. set the config). See something like: 
https://github.com/actions/checkout#push-a-commit-using-the-built-in-token

```
  build:
    runs-on: ubuntu-latest
    steps:
      - run: |
          # do some yq stuff here
          git config user.name github-actions
          git config user.email github-actions@github.com
          git add .
          git commit -m "some changes"
    # follow up with the create PR action
```

Reply:
Or perhaps not? Perhaps this action below does actually do it all?

Reply:
I think it does it all. The v4 api version allows that. I lost the link to the tutorial I followed but I will add it once i find it again